addon/components/nypr-story/main-image.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../../templates/components/nypr-story/main-image';
+
+export default Ember.Component.extend({
+  layout,
+  captionOpen: false,
+
+  didRender() {
+    this.$().imagesLoaded().progress((i, image) => {
+      Ember.run(() => {
+        image.img.classList.add('is-loaded');
+      });
+    });
+  },
+
+  actions: {
+    toggleCaption(){
+      if (this.get("captionOpen")){
```

Comment:
you can use `this.toggleProperty('captionOpen')` here instead of all this boilerplate